app/src/fdroid/java/org/fossasia/openevent/general/settings/AcknowledgementDecider.kt

```diff
@@ -0,0 +1,10 @@
+package org.fossasia.openevent.general.settings
+
+import android.content.Context
+
+object AcknowledgementDecider {
+
+    fun decide(context: Context) {
+       // don't show acknowledgements in fdroid
+    }
+}
```

Comment:
Add a method called `shouldShowAcknowledgement` and make it return true and false in the separate component

And use it to hide the acknowledgement